src/components/layout/Header.tsx

```diff
@@ -24,35 +60,91 @@ const Header = () => {
   }, []);
 
   return (
-    <WhiteBackNav className="container nav white-back" expand="lg" sticky="top">
-      <Navbar.Brand href="/">
-        <Logo height="60px" width="70px" />
-      </Navbar.Brand>
-      <Navbar.Toggle aria-controls="basic-navbar-nav" />
-      <Navbar.Collapse id="basic-navbar-nav">
-        <Nav className="ml-auto">
-          <Nav.Link as={Link} to="/">
-            HOME
-          </Nav.Link>
-          <Nav.Link as={Link} to="/who-we-are">
-            WHO WE ARE
-          </Nav.Link>
-          <Nav.Link as={Link} to="/donate">
-            DONATE
-          </Nav.Link>
-          {showSession && (
-            <Nav.Link as={Link} to="/logout">
-              LOGOUT
+    <>
+      <BlobRow>
+        {blobs.map(blobs => (
+          <img
+            alt=""
+            key={blobs.id}
+            src={blobs.blob}
+            style={{
+              inset: blobs.inset,
+              opacity: blobs.opacity,
+              transform: blobs.transform
+            }}
+          />
+        ))}
+      </BlobRow>
+      <WhiteBackNav
+        className="container nav white-back"
+        expand="lg"
+        sticky="top"
+      >
+        <Navbar.Brand href="/">
+          <Logo height="52px" width="62px" />
+        </Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="ml-auto">
+            <Nav.Link as={Link} to="/">
+              HOME
             </Nav.Link>
-          )}
-        </Nav>
-      </Navbar.Collapse>
-    </WhiteBackNav>
+            <Nav.Link as={Link} to="/who-we-are">
+              WHO WE ARE
+            </Nav.Link>
+            <Nav.Link as={Link} to="/donate">
+              DONATE
+            </Nav.Link>
+            <LogButton>
+              {!showSession && (
+                <Nav.Link as={Link} to="/login">
+                  LOG IN / SIGN UP
+                </Nav.Link>
+              )}
+              {showSession && (
+                <Nav.Link as={Link} to="/logout">
+                  LOGOUT
+                </Nav.Link>
+              )}
+            </LogButton>
+          </Nav>
+        </Navbar.Collapse>
+      </WhiteBackNav>
+    </>
   );
 };
 
 const WhiteBackNav = styled(Navbar)`
-  background-color: ${colors.bodyBg};
+  background-color: ${colors.white80a};
+  max-width: 100vw;
+  padding: 0.875rem 6.15vw;
```

Comment:
Were these values taken from the mock? We should probably try to just use rem because vw here can be a bit finicky in different sizes. Have you tried seeing what this looks like at different resolutions?